srcpkgs/OTPClient/patches/fix-cflags.patch

```diff
@@ -1,17 +1,17 @@
-Index: OTPClient-2.4.2/CMakeLists.txt
+Index: OTPClient-2.4.4/CMakeLists.txt
 ===================================================================
---- OTPClient-2.4.2.orig/CMakeLists.txt
-+++ OTPClient-2.4.2/CMakeLists.txt
+--- OTPClient-2.4.4.orig/CMakeLists.txt
++++ OTPClient-2.4.4/CMakeLists.txt
```

Comment:
The thing with the prefix is that it shouldn't require being updated for every version number, since the first path is "ignored" :)

Reply:
Oh right I forgot it is -p1 :)